Def 22 photos not corresponding with design

nomarchia opened the pull request.

nomarchia committed:
[def-22] Photos not corresponding with design:

Added icons from figma and used them instead of fa-icons;
Added name of the photo to photo-card;
Added method getName() to Photo.java;
Corrected width of photos area and size of photo-cards;
Checkbox not doesn't overlap the photo;
Changed bottom photo-card panel according to figma design;
Removed redundant comments from StaticPageServlet;

nomarchia committed:
[def-22] Photos not corresponding with design:

Replaced photo button icons with .svg instead of .png files;
Fixed bug with delete order confirmation popup;

OleksandrGubariev commented on src/main/webapp/WEB-INF/templates/order.html:
```diff
@@ -189,16 +195,14 @@ <h2 class="error" align="center" th:if="${errorMessage!=null}" th:text="${errorM
-                        onclick="btnUserReselect.disabled=false; disabled=true; frmNew.submit();"
-                >
+                        onclick="btnUserReselect.disabled=false; disabled=true; frmNew.submit();">
                     Submit
```
я здесь внес изменения. сначала нужно смержить Ромину ветку.

OleksandrGubariev commented on src/main/webapp/WEB-INF/templates/order.html:
```diff
@@ -189,16 +195,14 @@ <h2 class="error" align="center" th:if="${errorMessage!=null}" th:text="${errorM
-                        onclick="btnUserSubmit.disabled=false; disabled=true;"
-                >
+                        onclick="btnUserSubmit.disabled=false; disabled=true;">
                     Re-select
```
я здесь внес изменения. сначала нужно смержить Ромину ветку.

Yuliia-Sh commented on src/main/java/com/photostudio/entity/photo/Photo.java:
```diff
@@ -11,4 +11,11 @@
+        String name = partsOfPath[partsOfPath.length - 1];
+
+        return name.substring(0, name.indexOf("."));
+    }
```
тест на этот метод есть?

Yuliia-Sh replied:
Если в качестве source просто имя файла - это будет работать?

nomarchia replied:
Убрал лишнюю часть и добавил тест на метод.

nomarchia committed:
[def-22] Photos not corresponding with design:

Edited method getName() of Photo.java class, removed redundant part;
Added test for getName() method;

nomarchia committed:
[def-22] Photos not corresponding with design:

Quickfix;

nomarchia committed:
Update application.properties
